data/records/cms-tools-ana-Run2011A.json

```diff
@@ -129,7 +129,7 @@
       "CMS"
     ],
     "license": {
-      "attribution": "GNU General Public License (GPL) version 3"
+      "attribution": "GPL-3.0-only"
```

Comment:
For example, in attribution it is not fully specific which license it was meant to be.

Reply:
Yes, this should be OK. However if we want to be very precise, you can check the license body that exists in the linked repository: https://github.com/cms-opendata-analyses/pattuples2011/blob/master/COPYING

Reply:
I think it is better to leave it with `-only`. It should be much easier to expand that sometime in the future if needed, than later revert the changes.